examples/onft721/test/foundry/MyONFT721.t.sol

```diff
@@ -0,0 +1,152 @@
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
These tests are not run.  You need to modify the "test" script in package.json to:

```
"test": "$npm_execpath run test:forge && $npm_execpath run test:hardhat",
```